ansible/data/molecule/requirements.txt

```diff
@@ -1,104 +1,110 @@
+boto3==1.21.33
+botocore==1.24.33
+bracex==2.2.1
 Cerberus==1.3.2
-certifi==2020.12.5
-cffi==1.14.4
-cfgv==3.1.0
+certifi==2021.10.8
+cffi==1.15.0
+cfgv==3.3.1
 chardet==4.0.0
-click==7.1.2
+charset-normalizer==2.0.12
+click==8.1.2
 click-completion==0.5.2
-click-help-colors==0.9
+click-help-colors==0.9.1
 colorama==0.4.4
 commonmark==0.9.1
-cookiecutter==1.7.2
+cookiecutter==1.7.3
 cryptography==3.3.2
-dataclasses==0.8
-distlib==0.3.0
-distro==1.5.0
-docker==4.4.1
+dataclasses==0.6
+distlib==0.3.4
+docker==5.0.3
 docker-pycreds==0.4.0
-docutils==0.15.2
-enrich==1.2.6
-entrypoints==0.3
+enrich==1.2.7
+entrypoints==0.4
 enum34==1.1.10
-fasteners==0.15
-filelock==3.0.12
-flake8==3.7.9
+fasteners==0.17.3
+filelock==3.6.0
+flake8==4.0.1
 future==0.18.2
 git-url-parse==1.2.2
-identify==1.4.11
-idna==2.10
-importlib-metadata==3.3.0
-importlib-resources==1.4.0
-Jinja2==2.11.3
+identify==2.4.12
+idna==3.3
+importlib-metadata==4.11.3
+importlib-resources==5.6.0
+iniconfig==1.1.1
+Jinja2==3.1.1
 jinja2-time==0.2.0
-jmespath==0.9.5
-lxml==4.6.5
-MarkupSafe==1.1.1
-mccabe==0.6.1
-molecule==3.2.1
-molecule-docker==0.3.3
-molecule-ec2==0.1
-molecule-podman==0.3.0
-monotonic==1.5
-more-itertools==8.2.0
-nodeenv==1.3.5
-packaging==20.8
-paramiko==2.7.2
-pathlib2==2.3.5
-pathspec==0.8.1
-pbr==5.4.4
+jmespath==1.0.0
+lxml==4.8.0
+MarkupSafe==2.1.1
+mccabe==0.6.0
+molecule==3.6.1
+molecule-docker==1.1.0
+molecule-ec2==0.4
+molecule-podman==1.1.0
+monotonic==1.6
+more-itertools==8.12.0
+nodeenv==1.6.0
+packaging==21.3
+pathlib2==2.3.7.post1
+pathspec==0.9.0
+pbr==5.8.1
 pexpect==4.8.0
-pluggy==0.13.1
+platformdirs==2.5.1
+pluggy==1.0.0
 poyo==0.5.0
-pre-commit==1.21.0
-psutil==5.7.0
-ptyprocess==0.6.0
-py==1.10.0
-pycodestyle==2.5.0
-pycparser==2.20
-pyflakes==2.1.1
-Pygments==2.7.4
-PyNaCl==1.4.0
+pre-commit==2.18.1
+psutil==5.9.0
+ptyprocess==0.7.0
+py==1.11.0
+pycodestyle==2.8.0
+pycparser==2.21
+pyflakes==2.4.0
+Pygments==2.11.2
 pyparsing==2.4.7
-pytest==5.4.1
-python-dateutil==2.8.1
+pyqtgraph==0.12.4
+pytest==7.1.1
+pytest-testinfra==6.6.0
 python-gilt==1.2.3
-python-slugify==4.0.1
+python-slugify==6.1.1
 PyYAML==5.4.1
-requests==2.25.1
-rich==9.6.1
-ruamel.yaml==0.16.10
-ruamel.yaml.clib==0.2.0
-s3transfer==0.3.3
+requests==2.27.1
+resolvelib==0.5.4
+rich==12.1.0
+ruamel.yaml==0.17.21
+ruamel.yaml.clib==0.2.6
+s3transfer==0.5.2
 selinux==0.2.1
-sh==1.12.14
-shellingham==1.3.2
+sh==1.14.2
+shellingham==1.4.0
 six==1.15.0
-subprocess-tee==0.2.0
-tabulate==0.8.6
-testinfra==5.0.0
+subprocess-tee==0.3.5
+tabulate==0.8.9
+tenacity==8.0.1
+testinfra==6.0.0
+testresources==2.0.1
 text-unidecode==1.3
-toml==0.10.0
+toml==0.10.2
+tomli==2.0.1
 tree-format==0.1.2
-typing-extensions==3.7.4.3
-urllib3==1.26.5
-virtualenv==20.0.12
-wcwidth==0.1.8
-websocket-client==0.57.0
+typing-extensions==4.1.1
+urllib3==1.26.9
+virtualenv==20.14.0
+wcmatch==8.3
+wcwidth==0.2.5
+websocket-client==1.3.2
 whichcraft==0.6.1
-yamllint==1.25.0
-zipp==3.4.0
-pyqtgraph==0.11.1
```

Comment:
I think one of the plugins depends on pyqtgraph==0.11.1

Reply:
This wouldn't affect that, this is for molecule tests (this PR is also just open as I needed to test those molecule tests) Thanks for reviewing though :)